config.py

```diff
@@ -80,6 +81,9 @@ class BaseConfig:
     #: Where to store user uploads (PDFs, images, etc.).
     UPLOAD_FOLDER = _env("FLASK_UPLOAD_FOLDER")
 
+    #: Logger setup
+    LOG_LEVEL = 0
```

Comment:
Let's use `INFO` here. My view is that we should minimize production risks/complexity by using default settings that are as close to prod as possible.